core/core.gradle.kts

```diff
@@ -15,15 +15,21 @@ application {
 val os = osdetector.classifier.replace("osx", "macosx").replace("x86_32", "x86")
 val arch = osdetector.arch.replace("x86_64", "x64")
 
+val withCuda = project.hasProperty("cuda") || project.hasProperty("WITH_CUDA")
+
+if (withCuda) {
+    version = "$version-cuda"
+}
```

Comment:
Is the plan to just build a different version of GRIP with CUDA support?

Reply:
Yeah. CUDA-enabled OpenCV will fail to load if there's no CUDA runtime available, and I'm not sure if it's possible to trick JavaCPP into loading the non-CUDA version at runtime

Reply:
Classloader hackyness? A thought you could experiment with after this PR is merged.